pegasus/sites.v3/hourofcode.com/i18n/public/af/how-to/companies.md.partial

```diff
@@ -68,17 +68,17 @@ For more guidance on volunteering and sample messaging to get your company excit
 
 If you want to go the extra mile, your company can also host an Hour of Code event. There is no specific formula for hosting an event so we encourage you to be creative! Given that our staff size is small (but mighty!), Code.org does not have a team dedicated to coordinating Hour of Code events. However, here are some tips to get started.
 
-### 1. Watch this how-to video <iframe width="500" height="255" src="//www.youtube.com/embed/SrnvvWDm73k" frameborder="0" allowfullscreen></iframe> 
+### ۱. این ویدئو چگونه را تماشا کنید <iframe width="500" height="255" src="//www.youtube.com/embed/SrnvvWDm73k" frameborder="0" allowfullscreen></iframe> 
 
 ### 2. Determine a date, format, and location for your event
 
-We’ve seen events that are [virtual](https://hourofcode.com/us/how-to/virtual) or in-person, at the company office or held in the community, and for students of all ages! You can also take a look at what other [corporate partners](https://medium.com/@codeorg/amazon-microsoft-google-vista-and-more-rally-to-bring-the-hour-of-code-to-students-worldwide-4641325542cf) and [donors](https://medium.com/@codeorg/how-code-orgs-corporate-supporters-helped-spread-the-love-for-2019-s-hour-of-code-73a3c088f10f) have done in the past for some inspiring ideas.
+ما رویدادهایی را دیده ایم که م [جازی](https://hourofcode.com/us/how-to/virtual) یا حضوری، در دفتر شرکت یا در جامعه برگزار می شود و برای دانش آموزان در تمام سنین! همچنین می توانید نگاهی به سایر شرکای شرکتی</a> و اه داکنندگان در گذشته [برای](https://medium.com/@codeorg/how-code-orgs-corporate-supporters-helped-spread-the-love-for-2019-s-hour-of-code-73a3c088f10f) برخی ایده های الهام بخش انجام داده اند نگاهی بیندازید.</p> 
```

Comment:
With RL languages I can't really tell if the formatting of links is correct or not.